package.json

```diff
@@ -24,15 +24,15 @@
         "build": "yarn asbuild:untouched && yarn asbuild:optimized && yarn build:lib",
         "asbuild:untouched": "asc assembly/index.ts -b build/untouched.wasm -t build/untouched.wat --runtime full --validate --debug",
         "asbuild:optimized": "asc assembly/index.ts -b build/optimized.wasm -t build/optimized.wat --runtime full --validate -O3 --noAssert",
-        "build:lib": "babel src -d lib",
+        "build:lib": "babel src -d lib --copy-files",
         "build:web": "webpack --mode production --entry ./index.js --output ./dist/as-sha256.min.js",
         "test": "yarn test:as",
         "test:ci": "yarn test:as-ci",
         "test:as": "asp --nortrace --verbose",
         "test:as-ci": "asp --nortrace 2> /dev/null"
     },
     "dependencies": {
-        "assemblyscript": "https://github.com/AssemblyScript/assemblyscript.git"
+        "assemblyscript": "https://github.com/AssemblyScript/assemblyscript.git#5b510571f6ebfc5530d85412005059e4cf961a97"
```

Comment:
Why are you targeting a commit?

Reply:
if they introduce breaking change to master it will break all apps depending on this library. Until they release and tag stable version. We should check if sha works on targeted commit

Reply:
oh touché

Reply:
Yeah, that has happened to be before when they upgraded to the new loader. Good call !